easybuild/easyconfigs/d/DFT-D4/DFT-D4-3.2.0-intel-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'MesonNinja'
+
+name = 'DFT-D4'
+version = '3.2.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.chemie.uni-bonn.de/pctc/mulliken-center/software/dftd4'
+description = """Generally Applicable Atomic-Charge Dependent London Dispersion Correction."""
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+
+source_urls = ['https://github.com/dftd4/dftd4/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+patches = ['DFT-D4-%(version)s-remove_module_id.patch']
+checksums = [
+    '9874db9e2329519db258dd75ee7ce7c97947f975b00087ba5fdf9a28741088f1',  # v3.2.0.tar.gz
+    '8c3c81338cb57972580e4cf3db307aa2e44b8b3f6d1ba7ae24fa9d807490a93b',  # DFT-D4-3.2.0-remove_module_id.patch
+]
+
+builddependencies = [
+    ('Ninja', '1.9.0'),
+    ('Meson', '0.58.0', versionsuffix),
```

Comment:
@Micket Let's change this to `0.59.1`, since we already have an easyconfig for that since #13910

Change made in https://github.com/Micket/easybuild-easyconfigs/pull/8